### ISSUE TITLE
Tests without asserts

### ISSUE THREAD

**Issue by benmccann** [jtablesaw/tablesaw#436]
Codacy found that we have quite a few tests that don't assert anything:

https://app.codacy.com/app/jtablesaw/tablesaw/issues?bid=5238430&step=0&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOlsiQ29kZVN0eWxlIl19LHsiaWQiOiJMZXZlbCIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiUGF0dGVybiIsInZhbHVlcyI6WzU1NzVdfSx7ImlkIjoiQXV0aG9yIiwidmFsdWVzIjpbbnVsbF19LHsidmFsdWVzIjpbXX1d

**Comment by lwhite1** [comment]
I'm sure most of these are legit, but a number are cases where there is an assert in a private method in the test class.

**Comment by benmccann** [comment]
Yeah. I found that we can filter out the false positives if we make the private method start with `assert`. I saw in the past that we had some named `checkSomething` and we could rename the method to `assertSomething` to make it clearer which were legitimate

**Comment by lwhite1** [comment]
Cool. That’s helpful.
